.env.local

```diff
@@ -5,3 +5,6 @@ NEXT_PUBLIC_URL_IO=https://io.italia.it
 NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L1=https://www.spid.gov.it/SpidL1
 NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L2=https://www.spid.gov.it/SpidL2
 NEXT_PUBLIC_JWT_SPID_LEVEL_VALUE_L3=https://www.spid.gov.it/SpidL3
+
+# MOCK API FOR TEST ENVIRONMENT
+NEXT_PUBLIC_API_MOCK_WEB_PROFILE=false
```

Comment:
Is this related with tests in this PR?

Reply:
seems to be something related to something else, sorry, let's remove it